samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,126 @@ nRF5340: Network core bootloader
 
-The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
-This sample is included automatically if the application in the nRF5340 network core has the :option:`CONFIG_SECURE_BOOT` option set.
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+You can include the sample in your multi-image build as follows:
 
-A new set of firmware update files is generated by the build system when this image is included in a multi-image build.
-These files are identical to those described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+* Enabling the :option:`CONFIG_SECURE_BOOT` option in the application that has to run on the nRF5340 network core.
+  The build system will include this sample automatically.
+* Enabling the :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53` options in the application that has to run on the application core.
+  The build system will enable the :option:`CONFIG_SECURE_BOOT` option automatically for the network core application.
 
-Testing
-=======
+When you include this image in a multi-image build, the build system generates a new set of firmware update files
+These files match the ones described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+
+.. |sample path| replace:: :file:`samples/nrf5340/netboot/`
+
+.. include:: /includes/build_and_run.txt
+See also the following pages:
+
+* :ref:`ug_nrf5340_building` for additional details on building samples on the nRF5340.
+* :ref:`configure_application` for additional details on configuring your application.
+
+See below for an example of a multi-image build that includes :ref:`peripheral_uart`, using either SEGGER Embedded Studio or the command line:
+
+.. tabs::
 
-To test the network core bootloader sample run the following commands:
+   .. group-tab:: SEGGER Embedded Studio
 
-#. |connect_terminal|
+      Follow the instructions linked above, enabling the following options:
 
-   Note that on the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+      * :option:`CONFIG_BOOTLOADER_MCUBOOT`:
 
-#. ``west build -b nrf5340dk_nrf5340_cpuapp -d build_netboot samples/bluetooth/peripheral_uart -f -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_PCD=y``
+        #. Select :guilabel:`Project` -> :guilabel:`Configure nRF Connect SDK project`.
+        #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` -> :guilabel:`Bootloader` and set :guilabel:`Use MCUboot` to enable :option:`CONFIG_BOOTLOADER_MCUBOOT`.
```

Comment:
This isn't where this option is located when I test it. Please check.

Reply:
fixed